docs/protocols/cota_witness.md

```diff
@@ -190,8 +205,8 @@ table TransactionProof {
 ```jsx
 # CoTA cell data structure
 data:
-    version: byte     // must be 0 or 1(V1)
-    smt_root: optional<byte32>
+    version: byte
+    smt_root: Option<byte32>
```

Comment:
I think byte32 is better and here is pseudo code, not real coding implementation

Reply:
updated